doc/source/user_guide/changelog/v0.5.0.rst

```diff
@@ -0,0 +1,64 @@
+.. _whatsnew_050:
+
+What's new in 0.5.0 (2 December 2021)
+-----------------------------------
+
+These are the changes in icepyx 0.5.0 See :ref:`release` for a full changelog
+including other versions of icepyx.
+
+
+New Features
+~~~~~~~~~~~~
+
+- Add ICESat-2 data read-in functionality (#222)
+    * includes read module and is2cat (catalog) module, example notebook, and docs
+    * basic tests file for read module validation functions
+    * add basic build catalog function to read data object
+    * add ability to get list of variables from a file
+    * add variables example notebook; trim variables module details out of subsetting example
+    * update examples from 2020 Hackweek tutorials
+- add github action to add binder badge to PRs (#229 and #233)
```

Comment:
I would probably classify this under 'maintenance' or 'other' since it's not a user-facing feature.